hooks/kubernetes_installer.py

```diff
@@ -20,43 +18,44 @@ def __init__(self, arch, version, kubernetes_file):
                         'kubectl': 'kubectl'}
         self.arch = arch
         self.version = version
-        self.kubernetes_file = kubernetes_file
+        self.kubernetes_file = path(kubernetes_file)
 
-    def install(self, output_dir='/opt/kubernetes'):
+    def install(self, output_dir=path('/opt/kubernetes')):
         """ Install kubernetes binary files from the tar file or gsutil. """
-        if os.path.isdir(output_dir):
-            # Remote old content to remain idempotent.
-            shutil.rmtree(output_dir)
-        kubernetes_repo = 'http://github.com/GoogleCloudPlatform/kubernetes'
-        git_clone = 'git clone {0} {1}'.format(kubernetes_repo, output_dir)
-        print(git_clone)
-        subprocess.check_call(git_clone.split())
+        bindir = output_dir / 'bin'
+        if bindir.exists():
+            for fp in bindir.files():
+                fp.remove()
```

Comment:
This logic is not equivalent.  The old code removes the "/opt/kubernetes" directory and this code only removes "/opt/kubernetes/bin".  We need to change this to remove the kubernetes directory that was cloned by git clone.